config.toml

```diff
@@ -13,6 +13,16 @@
 # directory that you are sharing to the printer through the USB Gadget
 files_directory = "/mnt/usb_share"
 
+[file_list]
+# The sort order for the file list in the frontend. Can be:
+#   * "alpha" - for Alphanumeric sort
+#   * "creation" - for creation date/time sort
+#sort_order = "creation"
+
+# Show or hide files.  
+# Must be "true" or "false" 
+#show_hidden = "true"
```

Comment:
TOML files support booleans:

```toml
show_hidden = true
```

see https://learnxinyminutes.com/docs/toml/

can we use that instead of a string?

Reply:
Tried that. TOML might support booleans, but only all lowercase. Python wants the first character uppercase, so this doesn't work.